third-party/lib/nuklear/patch.txt

```diff
@@ -0,0 +1,12 @@
+diff -aur -x configure ../nuklear-master-orig/nuklear-master/nuklear.h nuklear-master/nuklear.h
+--- ../nuklear-master-orig/nuklear-master/nuklear.h	2017-01-25 01:06:59.000000000 +0300
++++ nuklear-master/nuklear.h	2017-02-02 19:47:27.613793130 +0300
+@@ -272,7 +272,7 @@
```

Comment:
Ops, You are right!
I suggested a temporary decision. I clone 'nuklear' to our repo and change link.
Now we have some troubles with google disk so I preferred  this variant.